### PR TITLE
chore(user-notification): Enable fetch client circuit breaker

### PR DESCRIPTION
**Pull request by rafnarnason**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by eirikurn — APPROVED** [review]

File: workspace.json [hunk 1]
```diff
@@ -92,7 +92,9 @@
               "inspect": false
             }
           },
-          "outputs": ["{options.outputPath}"]
+          "outputs": [
```
Comment (darrikonn) [inline workspace.json:95]: Why the formatting though?

**Review by darrikonn — CHANGES_REQUESTED** [review]
^

File: workspace.json (hunk 1 above)
Comment (rafnarnason) [reply workspace.json]: weird behaviour - slightly less dramatic changes via yarn format but still a minor change in workspace - not sure how this is happening...

**Review by kristofer-lp8f — APPROVED** [review]
LGTM 👍

**Comment by kristofer-lp8f** [comment]
@darrikonn, Requested changes fixed. Do you approve now?

**Review by darrikonn — APPROVED** [review]

**Pull request merged**